about.php

```diff
@@ -0,0 +1,211 @@
+<?php include'head.php';?>
+
+<div id="pcourses">
+    <span>About us</span>

```

Comment:
This looks like a title to me. Semantically we should use `<h1>`, `<h2>`, `<h3>`. 
This helps search machines to guess the semantic priorities of your text. :)